src/controllers/user_controller.cr

```diff
@@ -0,0 +1,52 @@
+require "../repositories/*"
+require "kemal"
+
+module BrokenCrystals
+    module UserController
+        PREFIX = "users"
+        mutableRepo = MutableRepository.new "users"
```

Comment:
underscore case for variable name `mutable_repo`
as per https://crystal-lang.org/reference/conventions/coding_style.html

Reply:
ameba linted

Reply:
@sixaphone yeha, there was a bug, ameba won't lie no more